packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -109,7 +109,7 @@ contract OPContractsManager is ISemver {
         address disputeGameFactoryImpl;
         address anchorStateRegistryImpl;
         address delayedWETHImpl;
-        address mipsImpl;
+        address mips64Impl;
```

Comment:
MIPS is a singleton for all chains, so putting it in this struct feels a bit misleading